resources/kubernetes/operator/helm/values.yaml

```diff
@@ -230,6 +245,16 @@ collectors:
             - key: service.name
               from_attribute: app.label.component
               action: insert
+            - key: service.name
+              from_attribute: k8s.container.name
+              action: insert
+            - key: app.label.component
```

Comment:
we had those 2 fields also ingested, additionally it was shown as 4 fields as we use aliases
<img width="385" alt="Screenshot 2024-10-10 at 18 34 48" src="https://github.com/user-attachments/assets/cf532163-1a2c-4925-9053-6b93dcfb4993">

Reply:
I reckon this is expected atm, they are ingested as "resource.attributes" and the otel plugin makes them available at top level. This is accepted for 8.16, see https://github.com/elastic/opentelemetry-dev/issues/522

Reply:
If that is the case, should we rollback these attributes removal?

Reply:
what I meant - we have the same information repeated twice: once as `resource.attributes.app.label.version` and second time as `resource.attributes.service.version`
the same for the service.name - once `resource.attributes.app.label.component` and `resource.attributes.service.name`.
`resource.attributes.app.label.*` is redundant and used more like a temporary variable

Reply:
Got it, thanks for the explanation.

Reply:
I think `resource.attributes.app.label.*` on its own do not provide any value and we don't use it. @gizas maybe I am missing smth?

Reply:
We will need to add:

- labels.app_kubernetes_io/name
- labels.app_kubernetes_io/version

(based on the discussion here: https://github.com/elastic/opentelemetry-dev/issues/496#issuecomment-2406802932)

Can we add those?

Reply:
@gizas but it is done already - current behavior is repeating the pipeline.
pipeline:
```
description: Pipeline for Kubernetes container logs
processors:
  - set:
      field: service.name
      copy_from: kubernetes.labels.app_kubernetes_io/name
      ignore_empty_value: true
  - set:
      field: service.name
      copy_from: kubernetes.container.name
      override: false
      ignore_empty_value: true
  - set:
      field: service.version
      copy_from: kubernetes.labels.app_kubernetes_io/version
      ignore_empty_value: true
```

otel config:
```
resource/k8s:
          attributes:
            - key: service.name
              from_attribute: app.label.component
              action: insert                           <-----   step 1. insert if not present service.name based on `app.kubernetes.io/component`
            - key: service.name
              from_attribute: k8s.container.name
              action: insert                         <-----   step 2. insert if not present service.name based on container name
            - key: app.label.component
              action: delete                        <----- step 3. remove redundant field app.label.component
            - key: service.version
              from_attribute: app.label.version
              action: insert                         <----- step 4.  insert if not present service.version based on `app.kubernetes.io/version`
            - key: app.label.version
              action: delete                        <----- step 5. remove redundant field app.label.version
```